pkg/commands/drift/drift.go

```diff
@@ -317,26 +318,11 @@ func (d *IAMDriftDetector) URI(i *assetinventory.AssetIAM) string {
 	}
 }
 
-// differenceMap finds the keys located in the left map that are missing in the right map.
-// We return a set so that we can do future comparisons easily with the result.
-func differenceMap(left, right map[string]*assetinventory.AssetIAM) map[string]struct{} {
-	found := make(map[string]struct{})
-	for key := range left {
-		if _, f := right[key]; !f {
-			found[key] = struct{}{}
-		}
-	}
-	return found
-}
-
-// differenceSet finds the keys located in the left set that are missing in the right set.
-// We return a set so that we can do future comparisons easily with the result.
-func differenceSet(left, right map[string]struct{}) map[string]struct{} {
-	found := make(map[string]struct{})
-	for key := range left {
-		if _, f := right[key]; !f {
-			found[key] = struct{}{}
-		}
+// KeySet returns the keys in the map.
+func KeySet[K comparable, V any](m map[K]V) map[K]struct{} {
+	keys := make(map[K]struct{}, len(m))
+	for k := range m {
+		keys[k] = struct{}{}
 	}
-	return found
```

Comment:
Previously you were sorting the keys - do you want to do that here? Also https://pkg.go.dev/golang.org/x/exp/maps#Keys might be easier (also unsorted).

Reply:
I am using Keys to replace it now everywhere. Also sorted the outputs.